app/scripts/blocker_general.js

```diff
@@ -102,6 +102,22 @@ export class Blocker {
         }
         addBlocker(nodeConfigurations);
     }
+
+    watchPageForMutations() {
+      var self = this;
+      var mutationObserver = new MutationObserver(function(mutations) {
+        for(var i=0; i<mutations.length; ++i) {
+            // look through all added nodes of this mutation
+            for(var j=0; j<mutations[i].addedNodes.length; ++j) {
+                self.modifyContent(mutations[i].addedNodes[j]);
```

Comment:
Could you deactivate the observer while modifying the content? Maybe you coul collect the changes and run  `modifyContent` as seldom as possible?

Reply:
What do you think about that idea (https://github.com/dieparteidiepartei/afd-blocker-plugin/pull/3/commits/df5973b0fb93404acf01d97416226300b7dfdf34) as I don't want to loose any changes?

Reply:
Collecting the changes shouldn't make a difference as it only searchs for the match in the added elements

Reply:
I've just tested the code - the problem with this Mutation oberver is, that it fires very often and for every little change. I don't want to create a massive performance problem for users.

I see 2 possible solutions:
1) Don't use a mutation observer and watch for click events on "load more" and "pager" elements
2) "Throttle" the page changes (e.g.: run changes only after 2 seconds or so...)

Reply:
All these events are usually fired in one batch (outside of the mutations loop). What do you think about collecting them and then doing all changes at once? I will try it and notify you when I'm done.

Reply:
I [changed](https://github.com/dieparteidiepartei/afd-blocker-plugin/pull/3/commits/acca1e3e930a0f5a4ef3189a54d11068625098aa) it now. But I don't think it's way better.

Reply:
The XPath was absolute. I [changed](https://github.com/dieparteidiepartei/afd-blocker-plugin/pull/3/commits/b07899bdd3ce2a8f711435699aede30ed643742a) it and I think it should be better now.

Reply:
Okay - looks good now :) I'll comment on a few minor issues

Reply:
And thanks for big PR btw. :)